app/javascript/src/components/Dashboard/Notes/MenuBar.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
+import { Typography } from "neetoui";
+import { MenuBar as NeetoUIMenuBar } from "neetoui/layouts";
+
+const MenuBar = ({ showMenu }) => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <div className="flex">
+      <NeetoUIMenuBar showMenu={showMenu} title="Notes">
+        <NeetoUIMenuBar.Block active count={13} label="All" />
+        <NeetoUIMenuBar.Block count={2} label="Users" />
+        <NeetoUIMenuBar.Block count={7} label="Leads" />
+        <NeetoUIMenuBar.Block count={4} label="Visitors" />
+        <NeetoUIMenuBar.SubTitle
+          iconProps={[
+            {
+              icon: Search,
+              onClick: () => setIsSearchCollapsed(!isSearchCollapsed),
```

Comment:
https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

Reply:
Fixed with #20 

@yedhink The above code was taken from [NeetoUI/MenuBar](https://neetoui.onrender.com/?path=/docs/layouts-menubar--menu-bar-story) example. Should we update this there too ?

<img width="629" alt="Screenshot 2022-09-14 at 3 01 49 PM" src="https://user-images.githubusercontent.com/44225503/190118329-e5c8f182-9238-41c6-96a4-7a6e6b01dcea.png">

Reply:
@vishnuMohanan01 Go ahead and raise an issue for this in https://github.com/bigbinary/neeto-ui/issues and mention me in it.